### PR TITLE
SpreadSpectrum-Disable

### PR DESCRIPTION
**Pull request by andreyv1978**
- Created BaseIpcLib
- Sideband Interface picked from
    tianocore/edk2-platforms
    branch: devel-IntelAtomProcessorE3900
    commit: 181f9e6c6ccde6e3fa62278b3a8b39cfb5844a7c
- IPC Interface picked from
    tianocore/edk2-platforms
    branch: devel-IntelAtomProcessorE3900
    commit: 181f9e6c6ccde6e3fa62278b3a8b39cfb5844a7c
- Updated Stage1BBoardInitLib.C with a test function

Signed-off-by: Andrey Vinokurtsev <avinok@gmail.com>

### CODE REVIEW

**Comment by andreyv1978** [comment]
Created BaseIpcLib for future use. I am guessing noone actually needs it now. Therefore, would happy to keep it not used. I do not want to add an extra configuration for my board or any other boards.
I've not found an easy way to sort our the problem with SSC branch, can not update code there. Created a new branch, with a hope it would work for updates and commits

File: Platform/ApollolakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c [hunk 1]
```diff
@@ -1699,6 +1703,112 @@ PlatformFeaturesInit (
...
 
+/**
+    USB3, PCie, SATA, eDP, DP, eMMC, SD and SDIO SSC
+    Partially implemented PeiHighSpeedSerialInterfaceSSCInit and PeiDDRSSCInit
+    to disable SSC in LCPLL_CTRL_1 and LJ1PLL_CTRL
+**/
+STATIC
+VOID
+EFIAPI
+SetSSCDisable (
```
Comment (mauricema) [inline Platform/ApollolakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c:1714]: According to coding style,  you need to Use  SetSscDisble ()

File: Platform/ApollolakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c [hunk 2]
```diff
@@ -1699,6 +1703,112 @@ PlatformFeaturesInit (
...
+  MicroSecondDelay (1000);
+
+  //
+  // set the ssc_en to Disable!
+  //
+  WBuf.LJ1PLL_CTRL_1.Fields.ssc_en = SSC_DISABLE;
+  WBuf.LJ1PLL_CTRL_1.Fields.ssc_en_ovr = SSC_DISABLE;
+  Status = IpcSendCommandEx (IPC_CMD_ID_EMI_RFI_SUPPORT, IPC_SUBCMD_ID_SSC_APPLY_NOW, &WBuf, BufferSize);
+  if(EFI_ERROR (Status))
+  {
```
Comment (mauricema) [inline Platform/ApollolakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c:1781]: Accoring to coding style, the "{" needs to be at the end of line 1780.

**Comment by andreyv1978** [comment]
closed in favor of : https://github.com/slimbootloader/slimbootloader/pull/742

**Pull request closed without merging**